src/lib/data.ts

```diff
@@ -8,45 +8,159 @@ export const data = {
 					x: ['2022-10-02', '2022-10-08'],
 					y: 'Ender',
 					label: 'Job 1234 - Part A 500 total count 98% efficient     300 good count',
-					tooltip: [
-						['category', 'info'],
-						['category', 'info']
-					]
+					tooltip:
+					"<style>tr:nth-child(even) {\n" +
+						"  background-color: gray;\n" +
+						"}" +
+						"tr:nth-child(odd) {\n" +
+						"  background-color: darkgray;\n" +
+						"}"+
+						"</style>" +
+						"<table style='background-color: black'>\n" +
+						"  <tr>\n" +
+						"    <th>Company</th>\n" +
+						"    <th>Contact</th>\n" +
+						"    <th>Country</th>\n" +
+						"  </tr>\n" +
+						"  <tr>\n" +
+						"    <td>Alfreds Futterkiste</td>\n" +
+						"    <td>Maria Anders</td>\n" +
+						"    <td>Germany</td>\n" +
+						"  </tr>\n" +
+						"  <tr>\n" +
+						"    <td>Centro comercial Moctezuma</td>\n" +
+						"    <td>Francisco Chang</td>\n" +
+						"    <td>Mexico</td>\n" +
+						"  </tr>\n" +
+						"</table>"
 				},
```

Comment:
you can use the back tick to do a multiline string so you don't need the newline characters or concatenation

`
<style>
  tr:nth-child(even) {
    background-color: gray;
  }
  tr:nth-child(odd) {
    background-color: darkgray;
  }
</style>
<table style='background-color: black'>
  <tr>
    <th>Company</th>
    <th>Contact</th>
    <th>Country</th>
  </tr>
  <tr>
    <td>Alfreds Futterkiste</td>
    <td>Maria Anders</td>
    <td>Germany</td>
  </tr>
  <tr>
    <td>Centro comercial Moctezuma</td>
    <td>Francisco Chang</td>
    <td>Mexico</td>
  </tr>
</table>`